### ISSUE TITLE
How can i make a whitelist?

### ISSUE THREAD

**Issue by luluxiu1994** [Calamari-OCR/calamari#57]
i notice that the engine can use a whitelist_files to protect some characters not remove during finetuning.I load a pre-trained model and make a txt file list characters.After training,i found the model forget the characters in pre-trained model,it seems the whitelist i made dosen't work.Could you give me a example about the whitelist_files?(better a txt file with some characters)

**Comment by ChWick** [comment]
The white list protects its characters only very roughly. E. g. if you have a german model containing the German umlauts ÄÖÜ, but the actual new model sees a lot of AOUs, the umlauts may get mostly forgotten, cause during the training of the new model, especially when training for a long duration. Furthermore, a very different font compared to the finetuned model should decrease the accuracy of whitelisted characters. Thus, maybe you are already using the whiteliste correctly:
```
calamari-train --whitelist_files whitelist.txt ...
```
And the content of `whitelist.txt` simply a list of characters:
```
abcdefghijklmnopqrstuvwxzy....
```
You should get a log message of how many characters are new or deleted.

**Comment by luluxiu1994** [comment]
Thank you for answering my questions,i've got it.

**Comment by ChWick** [comment]
You are welcome!
